.github/workflows/docs.yml

```diff
@@ -15,6 +15,6 @@ jobs:
           node-version: "12.x"
       - run: npm ci
       - run: npm run build
-      - run: node script/publish-docs
```

Comment:
Here's the build failure:

```
# Exit if a command returns a non-zero code
^

SyntaxError: Invalid or unexpected token
    at Module._compile (internal/modules/cjs/loader.js:891:18)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:991:10)
    at Module.load (internal/modules/cjs/loader.js:811:32)
    at Function.Module._load (internal/modules/cjs/loader.js:723:14)
    at Function.Module.runMain (internal/modules/cjs/loader.js:1043:10)
    at internal/main/run_main_module.js:17:11
```

Looks like Node trying to run the script doesn't like the use of `bash`-esque comments. Upon looking at the script, that makes perfect sense because it is `bash` 😅